docs/reference/api/docker_remote_api.md

```diff
@@ -116,6 +116,8 @@ This section lists each version from latest to oldest.  Each listing includes a
 
 [Docker Remote API v1.25](docker_remote_api_v1.25.md) documentation
 
+* `POST /containers/create` now takes `AutoRemove` in HostConfig, auto-removal will be done on daemon side.
```

Comment:
I think it should be fine that way; we can always improve later if someone has better suggestions